nbopen.py

```diff
@@ -20,7 +21,7 @@ def find_best_server(filename, profile='default'):
     if profile != 'default':
         warnings.warn("Jupyter doesn't have profiles")
         kwargs['profile'] = profile
-    servers = [si for si in notebookapp.list_running_servers(**kwargs) \
```

Comment:
Another minor style thing -- apologies! I forgot that I had made these little stylistic changes.